docs/customize/configure.md

```diff
@@ -0,0 +1,621 @@
+
+Enable or disable Self-Checkout feature endpoint.
+
+## Database Configuration
+
+### SQLALCHEMY_DATABASE_URI
+
+Database URI including user and password.
+
+```python
+SQLALCHEMY_DATABASE_URI = "postgresql+psycopg2://test:psw@localhost/ils"
+```
+
+## JSONSchemas Configuration
+
+### JSONSCHEMAS_HOST
+
+Hostname used in URLs for local JSONSchemas.
+
+```python
+JSONSCHEMAS_HOST = "127.0.0.1:5000"
+```
+
+## CSRF Configuration
+
+### REST_CSRF_ENABLED (True/False)
+
+Enable or disable CSRF protection for REST APIs.
+
+## CORS Configuration
+
+### REST_ENABLE_CORS (True/False)
+
+Enable or disable CORS support for REST APIs.
+
+### CORS_SEND_WILDCARD (True/False)
+
+Allow or disallow credentials to be sent with CORS requests.
+
+### CORS_SUPPORTS_CREDENTIALS (True/False)
+
+Exable or disable supporting credentials for CORS requests.
+
+## Flask Configuration
+
+### SECRET_KEY
+
+Secret key used for session management and CSRF protection.
+
+### MAX_CONTENT_LENGTH
+
+Maximum allowed size for uploaded files via form data.
+
+```python
+MAX_CONTENT_LENGTH = 100 * 1024 * 1024  # 100 MiB
+```
+
+### SESSION_COOKIE_SECURE (True/False)
+
+Set cookie with the secure flag by default.
+
+### SESSION_COOKIE_SAMESITE
+
+Set SameSite attribute for session cookies.
+
+```python
+SESSION_COOKIE_SAMESITE = "Lax"
+```
+
+### APP_ALLOWED_HOSTS
+
+List of allowed hosts for the application.
+
+```python
+APP_ALLOWED_HOSTS = ["localhost", "127.0.0.1"]
+```
+
+### SPA_HOST
+
+Host URL for Single Page Application.
+
+```python
+SPA_HOST = "https://127.0.0.1:3000"
+```
+
+### SPA_PATHS
+
+Paths and placeholders for Single Page Application routes.
+
+```python
+SPA_PATHS = dict(
+    literature="/literature/%(pid)s",
+    loan="/backoffice/loans",
+    patron="/backoffice/patrons",
+    profile="/profile",
+    search="/search?q=%(querystring)s",
+)
+```
+
+Here's the continuation and completion of the configuration file documentation in Markdown format:
+
+## Records REST Configuration
+
+### RECORDS_REST_MAX_RESULT_WINDOW
+
+Maximum result window for Records REST endpoints.
+
+```python
+RECORDS_REST_MAX_RESULT_WINDOW = 10000
+```
+
+### RECORDS_REST_VOCAB_MAX_RESULT_WINDOW
+
+Maximum result window for vocabulary Records REST endpoints.
+
+```python
+RECORDS_REST_VOCAB_MAX_RESULT_WINDOW = 500
+```
+
+### PIDSTORE_RECID_FIELD
+
+Field name in the PID store for the record identifier.
+
+```python
+PIDSTORE_RECID_FIELD = "pid"
+```
+
+### REST_MIMETYPE_QUERY_ARG_NAME
+
+Name of the URL argument to choose response serializer.
+
+```python
+REST_MIMETYPE_QUERY_ARG_NAME = "format"
+```
+
+### RECORDS_REST_ENDPOINTS
+
+Dictionary defining the endpoints for Records REST API.
+
+```python
+RECORDS_REST_ENDPOINTS = {
+    "docid": {
+        "pid_type": DOCUMENT_PID_TYPE,
+        "pid_minter": DOCUMENT_PID_MINTER,
+        "pid_fetcher": DOCUMENT_PID_FETCHER,
+        "search_class": DocumentSearch,
+        "record_class": Document,
+        "indexer_class": DocumentIndexer,
```

Comment:
I would leave these and explain that they can be modified if customisation is needed on the Record class behaviour or search class or indexer, but these require some advanced developer experience